apps/user-service/tsconfig.build.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "../../frontend/tsconfig.json",
```

Comment:
Create tsconfig package and consume from there, see reference in turborepo examples (how they use tsconfig across the repo)